src/main/java/com/dremio/exec/store/hbase/HBaseScanDrel.java

```diff
@@ -28,13 +28,14 @@
 import com.dremio.exec.planner.common.ScanRelBase;
 import com.dremio.exec.planner.logical.Rel;
 import com.dremio.exec.store.TableMetadata;
+import com.google.common.collect.ImmutableList;
 
 public class HBaseScanDrel extends ScanRelBase implements Rel {
 
   public HBaseScanDrel(RelOptCluster cluster, RelTraitSet traitSet, RelOptTable table,
       StoragePluginId pluginId, TableMetadata tableMetadata, List<SchemaPath> projectedColumns,
       double observedRowcountAdjustment) {
-    super(cluster, traitSet, table, pluginId, tableMetadata, projectedColumns, observedRowcountAdjustment);
+    super(cluster, traitSet, table, pluginId, tableMetadata, projectedColumns, observedRowcountAdjustment, ImmutableList.of());
```

Comment:
What is `ImmutableList.of()` supposed to be a list of?

A good practice is to declare a named variable and use that in the argument list.

For example (in pseudocode):

```
List<Parts> emptyListOfCarParts = new List<Parts>()
super(serialNumber, invoiceNumber, emptyListOfCarParts)
```

This way, when someone looks at the list of arguments being called, they have a better idea of what list you should be passing in.